editors/code/src/main.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
We should probably replace that console access with our log class (feel free to put a FIXME on this eslint comment for that)

Reply:
Removed the line, this was only there for some debugging purposes.